docs/src/main/asciidoc/opentelemetry.adoc

```diff
@@ -589,6 +589,19 @@ It can be activated by setting `quarkus.otel.traces.exporter=otlp` and include t
 implementation("io.opentelemetry:opentelemetry-exporter-otlp-common")
 implementation("io.opentelemetry:opentelemetry-exporter-otlp")
 ----
+[[opentelemetry-technologies]]
+== Technologies and Extensions using OpenTelemetry in Quarkus
+
+* https://quarkus.io/extensions/io.quarkus/quarkus-agroal[`agroal`]
+* https://quarkus.io/guides/grpc-getting-started[`gRPC`]
+* https://quarkus.io/extensions/io.quarkus/quarkus-jaxrs-client-reactive[`JAX-RS client reactive`]
+* https://quarkus.io/guides/resteasy[`RESTEasy Classic`]
+* https://quarkus.io/guides/resteasy-reactive[`RESTEasy Reactive`]
+* https://quarkus.io/guides/rest-client[`REST Client`]
+* https://quarkus.io/guides/scheduler[`scheduler`]
+* https://quarkus.io/guides/smallrye-graphql[`SmallRye GraphQL`]
+* https://quarkus.io/guides/kafka-reactive-getting-started[`SmallRye Reactive Messaging`]
```

Comment:
If it's the Kafka lib, we should be explicit on it.
|Aren't there more RM libs supporting it?